--- config.h.orig	2020-03-19 04:20:44.638571954 +0100
+++ config.h	2020-03-19 04:21:44.698778868 +0100
@@ -5,7 +5,7 @@
  *
  * font: see http://freedesktop.org/software/fontconfig/fontconfig-user.html
  */
-static char *font = "Liberation Mono:pixelsize=12:antialias=true:autohint=true";
+static char *font = "mono:size=9:antialias=true:autohint=true";
 static int borderpx = 2;
 
 /*
@@ -49,7 +49,7 @@
  * blinking timeout (set to 0 to disable blinking) for the terminal blinking
  * attribute.
  */
-static unsigned int blinktimeout = 800;
+static unsigned int blinktimeout = 0;
 
 /*
  * thickness of underline and bar cursors
